osmomath/exp2.go

```diff
@@ -93,8 +93,8 @@ func exp2ChebyshevRationalApprox(x BigDec) BigDec {
 	for i := 1; i < len(numeratorCoefficients13Param); i++ {
 		x_exp_i.MulMut(x)
 
-		h_x = h_x.Add(numeratorCoefficients13Param[i].Mul(x_exp_i))
-		p_x = p_x.Add(denominatorCoefficients13Param[i].Mul(x_exp_i))
+		h_x.AddMut(numeratorCoefficients13Param[i].Mul(x_exp_i))
+		p_x.AddMut(denominatorCoefficients13Param[i].Mul(x_exp_i))
```

Comment:
Please apply `QuoMut` at line 100 too

Reply:
applied, but why do we need it there? Does not a normal `Quo` suffice?

Reply:
There should be a a performance improvement with the mutative version